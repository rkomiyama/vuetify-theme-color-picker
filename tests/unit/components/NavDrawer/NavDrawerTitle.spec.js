import Vue from 'vue';
import Vuetify from 'vuetify';
import { shallowMount } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer';
import NavDrawerTitle from '@/components/NavDrawer/NavDrawerTitle.vue'

Vue.use(Vuetify)
let wrapper

describe('NavDrawerTitle.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(NavDrawerTitle)
  })

  afterEach(() => {
    wrapper.destroy()
  })

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  })

  it('NavDrawerTitle renders the same HTML', () => {
    const renderer = createRenderer();
    renderer.renderToString(wrapper.vm, (err, str) => {
    if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
})
