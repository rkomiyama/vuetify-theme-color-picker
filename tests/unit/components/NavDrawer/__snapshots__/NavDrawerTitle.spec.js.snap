// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavDrawerTitle.vue NavDrawerTitle renders the same HTML 1`] = `
<v-toolbar-title-stub class="headline text-uppercase py-2 hidden-sm-and-up">
  <v-flex-stub tag="div" class="d-flex align-center">
    <v-img-stub height="36px" maxWidth="36px" position="center center" src="" transition="fade-transition"></v-img-stub> <span>GitHubÂ </span> <span class="font-weight-light">PERSONAS</span>
  </v-flex-stub>
</v-toolbar-title-stub>
`;
